log/api.go

```diff
@@ -47,7 +52,14 @@ func WithLogger(logger Logger) Fields {
 func (f Fields) Chain(fieldses ...Fields) Fields {
 	merged := f.m
 	for _, fields := range fieldses {
-		merged = merged.Update(fields.m)
+		if merged.Has(configKey{}) && fields.m.Has(configKey{}) {
+			config1 := merged.MustGet(configKey{}).(frozen.Map)
+			config2 := fields.m.MustGet(configKey{}).(frozen.Map)
```

Comment:
If you're going to call `Has` then `MustGet`, you should just call `Get` instead.

Reply:
removed due to the new implementation